policyengine_uk/variables/gov/dfe/targeted_childcare_entitlement/targeted_childcare_entitlement.py

```diff
@@ -0,0 +1,16 @@
+from policyengine_uk.model_api import *
+
+
+class targeted_childcare_entitlement(Variable):
+    value_type = float
+    entity = Person
+    label = "targeted childcare entitlement amount per year"
```

Comment:
```suggestion
    label = "targeted childcare entitlement amount"
```

The time period is always implicit